### PR TITLE
Blog app - integration specs for views

### PR DESCRIPTION
**Pull request by Abenezer-Tilahun**
### Description
In this project, we  create integration tests for all of the views used in your project. 

### General requirements
- Make sure that there are [no linter errors](https://github.com/microverseinc/linters-config).
- Make sure that you used correct [Gitflow](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/git-github/articles/gitflow.md).
- Make sure that you documented your work [in a professional way](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md).

### Ruby requirements

- Follow our list of [best practices for Ruby](https://github.com/microverseinc/curriculum-ruby/blob/main/articles/ruby_best_practices.md).

### Project requirements

- [x] Use Capybara to write integration tests for each view in your project.
- [x] write the integration test for Login page: User index page: user show page: User post index page: Post show page:

### CODE REVIEW

File: spec/integration/login_spec.rb [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+      within 'form' do
+        fill_in 'Email', with: @user1.email
+        fill_in 'Password', with: @user1.password
+      end
+      click_button 'Log in'
+      expect(page).to have_content 'Signed in successfully'
+    end
+  end
+end
```
Comment (AnselemOdims) [inline spec/integration/login_spec.rb:39]: - [x] Please include test for redirects to the root page when a correct username and password is entered.

File: spec/integration/user_show_spec.rb [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+    expect(page).to have_content 'Testing4 with capybara'
+    expect(page).to_not have_content 'Testing1 with capybara'
+  end
+
+  scenario 'if redirect to the user_post_index page when you click on a see all posts.' do
+    first(:link, href: user_path(@user.id)).click
+    click_link('See all posts')
+    expect(current_path).to eq "/users/#{User.first.id}/posts"
+  end
+end
```
Comment (AnselemOdims) [inline spec/integration/user_show_spec.rb:53]: - [x] Please kindly include a test for the redirect to the post's show page when you click on a user's post.

**Review by AnselemOdims — CHANGES_REQUESTED** [review]
### Hi @Abenezer-Tilahun, 
You have been doing a tremendous job so far :ok_hand:
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 🎉
PR has a title and summary. :heavy_check_mark:
No linter errors :heavy_check_mark:

## Required Changes ♻️
- [ ] Please also check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Please leave any questions or comments in the PR if you have any doubts or unclarity! You can tag me using @AnselemOdims for a quicker response

**_Note: Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: spec/integration/login_spec.rb (hunk 1 above)
Comment (Abenezer-Tilahun) [reply spec/integration/login_spec.rb:39]: @AnselemOdims Thanks for review it and for feedback i will fix that.

File: spec/integration/user_show_spec.rb (hunk 2 above)
Comment (Abenezer-Tilahun) [reply spec/integration/user_show_spec.rb:53]: Done.

Comment (Whoistolu) [reply spec/integration/user_show_spec.rb:53]: Resolved 👍🏽

File: spec/integration/login_spec.rb (hunk 1 above)
Comment (Whoistolu) [reply spec/integration/login_spec.rb:39]: Resolved 👍🏽

**Review by Whoistolu — APPROVED** [review]
### Hi @Abenezer-Tilahun 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

# STATUS: APPROVED 🟢
**Congratulations! 🎉** 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @whoistolu in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**